src/examples/regulated-stablecoin/regulated-stablecoin.cabal

```diff
@@ -84,6 +83,7 @@ library
     , convex-coin-selection
     , convex-optics
     , convex-wallet
+    , hset
```

Comment:
Not sure about this, but seems we might be able to get away without `hset` as most types and helpers are reexported from the offchain lib's `ProgrammableTokens.OffChain.Env.Utils` module.